chapter_linear-networks/weight-decay.md

```diff
@@ -412,14 +413,16 @@ and this work becomes more routine.
 
 ```{.python .input  n=14}
 %%tab all
-if tab.selected('mxnet') or tab.selected('tensorflow'):    
+if tab.selected('mxnet'):
     model = WeightDecay(wd=3, lr=0.01)
+if tab.selected('tensorflow'):
+    model = WeightDecay(wd=0.3, lr=0.01)
```

Comment:
Is it possible to keep the wd the same for all the frameworks?

Reply:
For tensorflow, the wd is a L2 regularization rate. For pytorch and mxnet, the regularization rate is defined as wd \times lr. So, actually, setting wd of tensorflow to 0.03 is equivalent to setting it to 3 in pytorch and mxnet. But I found that setting it to 0.3 is more visible in the plot so I chose 0.3 instead of 0.03.

Reply:
"setting wd of tensorflow to 0.03 is equivalent to setting it to 3 in pytorch and mxnet"
Can you provide more context? Can you also add the explanations in the main text right above the code?

Reply:
I double checked the tensorflow implementation and I found that it is also l2 regularization so I changed the wd back to 3. It is the weight initialization that effects the plot so I added the normal initialization.